askama_parser/src/expr.rs

```diff
@@ -51,6 +51,12 @@ macro_rules! expr_prec_layer {
     }
 }
 
+#[derive(Clone, Debug, PartialEq)]
```

Comment:
.. this should be somewhere below `Expr`, probably just above `Suffix`.

Reply:
You should add a lint about this, that would make your reviews much simpler. I keep forgetting about how you prefer the definitions to come after their use. ^^'

Reply:
(yep, C/C++ syndrom is hitting hard haha)

Reply:
I don't think linting for this exists?

(Most contributors pick up on it more quickly than you have so far... 😛 )

Reply:
I'm horribly slow on style things. I'm too used to have them enforced by lints. I've been spoiled. :crying_cat_face: 

And no, you'd need to write either a check or write your own linter (I wrote a blog post on how you can do it [here](https://blog.guillaume-gomez.fr/articles/2024-01-18+Writing+your+own+Rust+linter) if you're interested :wink: ) for this to be checked in the CI. Shouldn't be too hard I think.

Reply:
Oh boy. More work on top of my clippy/rustdoc/rustc_codegen_gcc/browser-ui-test/sysinfo/rustc pattern matching works? :o

Well, if you're not in a hurry, why not. ^^'